docker/cassandra-operator/Makefile

```diff
@@ -2,7 +2,7 @@
 .PHONY: cassandra-operator
 cassandra-operator:
 	docker build \
-		-t $(DOCKER_REPO)cassandra-operator \
+		-t $(DOCKER_REPO)/cassandra-operator \
```

Comment:
Yep. `DOCKER_REPO` should contain the `/`.
Not sure how to enforce this, but some thoughts:
- have a check to ensure that `DOCKER_REPO` is either empty or ends in a `/`.
- write a function which correctly joins `DOCKER_REPO` with the image name, inserting a `/` if necessary.

The last option is probably the best as it would allow `DOCKER_REPO` to be specified with or without a training `/`.

Reply:
yeah I think its becoming clear that from an ergonomics perspective the slash in the env var is annoying. This nit shouldn't block the merge if we revert it, but @zegelin we should just have something that handles it (e.g. your function) so that it doesn't matter and it's easier